android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -1180,7 +1180,12 @@ public int addAcctMgrErrorNum(String url, String userName, String pwd)
                 throws RemoteException {
             AcctMgrRPCReply acctMgr = clientInterface.addAcctMgr(url, userName, pwd);
             if (acctMgr != null) {
-                return acctMgr.error_num;
+                if(acctMgr.messages.get(0).equals(getString(R.string.rpcreason_user_pass_problem))) {
```

Comment:
Here will be an issue if user has localized (non-English) Android client. This statement is valid for English Android client only.
To show to the user localized error message is a good idea, but it requires changes on server side. As far as we don't know what server will sends us as a message in response and there is no error code returned with message, it is safer to show to the user just exact the same text that contains in response without any comparisons.